### ISSUE TITLE
Syncopy does not keep track of its temp storage 

### ISSUE THREAD

**Issue by pantaray** [esi-neuroscience/syncopy#456]
**Describe the bug**
Syncopy does not correctly compute the size of its temporary storage directory. 

**To Reproduce**
```python
import syncopy as spy

Syncopy 2022.12

See https://syncopy.org for the online documentation.
For bug reports etc. please send an email to syncopy@esi-frankfurt.de

Logging to log directory '/cs/home/kajald/.spy/logs'.
Temporary storage directory set to '/cs/home/kajald/.spy/tmp_storage'

from syncopy.datatype.util import setup_storage
setup_storage()
(1.9789702035486698, 8)

```
However, 

```bash
du -h --max-depth=1 /cs/home/kajald/.spy/tmp_storage/
1.5G	/cs/home/kajald/.spy/tmp_storage/spy_fe54_7bb8e9a9
14G	/cs/home/kajald/.spy/tmp_storage/spy_ca09_b73e54ff
512	/cs/home/kajald/.spy/tmp_storage/spy_fe54_bf55d6f2
281G	/cs/home/kajald/.spy/tmp_storage/spy_fe54_6d6399c5
512	/cs/home/kajald/.spy/tmp_storage/spy_fe54_deffa14a
512	/cs/home/kajald/.spy/tmp_storage/spy_fe54_41496fd4
512	/cs/home/kajald/.spy/tmp_storage/spy_fe54_194bcf7e
301G	/cs/home/kajald/.spy/tmp_storage/
```

**Expected behavior**
Print a warning message if the size of the storage directory exceeds the pre-set limit (of 10GB, I think). 

**System Profile:**
 - OS: Linux (RHEL 8.2)

**Comment by tensionhead** [comment]
So we traced the problem to the `os.scandir()` call within `datataype.util.setup_storage`, which **does not** step into directories and hence does not count anything in the example presented above. Should be easy to fix!

**Comment by pantaray** [comment]
Great, thanks!

**Comment by dfsp-spirit** [comment]
We have fixed the reported issue (not merged yet), but I think this is not the only problem. We noticed 2 things:

* DJs dir on the cluster contains some temp files in `~/.spy/tmp_storage/` which are ~ 1.3 TB in size. This should not happen. The files are also stored in the directory form, not the file form, of storing Syncopy data.
* Way less important, there exist some directories named `ACME_<date>_<time>_<msecs>` in DJs user home (not in the Syncopy directory), e.g., `ACME_230307_150312_523756`. These directories are small though, so they not much of a concern right now, but we should still find out where they are coming from and what we can do to clean them up, as they look like temporary data.

**Comment by dfsp-spirit** [comment]
@KatharineShapcott Just to be save, could you check the size of your `~/.spy/` dir on the cluster (cs and gs), please? If you are not currently running an analysis, it is safe to delete everything in there.

**Comment by KatharineShapcott** [comment]
We actually added additional steps to our preprocessing pipeline to delete files in this folder to avoid these problems. Here's the code in case it's useful:

```
from syncopy import __storage__
files = glob.glob(__storage__+'/*', recursive=True)
for file in files:
    flagDelete = False
    fileAge = (time.time() - os.path.getmtime(file))/3600
    ext = os.path.splitext(file)[-1].lower()
    # Check if h5 and older than a day, delete.
    if (ext == ".h5") * (fileAge > 24):
        flagDelete = True
    # If very big, and older than 6 hours delete.
    elif (os.path.getsize(file) > 50 * 1024 ** 3) and (fileAge > 6):
        flagDelete = True
    if flagDelete:
        print(file+' will be removed')
        os.remove(file)

```

**Comment by tensionhead** [comment]
Addressed with #457
